.tx/config

```diff
@@ -0,0 +1,8 @@
+[main]
+host = https://www.transifex.com
+
+[stanford-openedx-xblocks.free-text-response]
```

Comment:
I've already created a Transifex repository for #42, could we please use that?

 - https://www.transifex.com/open-edx/xblocks/xblock-free-text-response/

This way, when I need to only invite translators (and I do that often) to a single project instead of many Transifex projects.

Reply:
Yes it's owned by edX and I'm also an admin so ping me if you needed anything.

We also would like to democratize it as much as possible so the Translators don't have to login into a dozen different account.